11ty/definitions/dude.md

```diff
@@ -0,0 +1,20 @@
+---
+title: dude
+slug: dude
+defined: true
+reading:
+  - text: Etymonline entry
+    href: https://www.etymonline.com/word/dude
+---
+
+colloquially, a man; a discourse marker expressing masculine solidarity; an expression of astonishment when used alone
+
+## Issues
+
+Dude was originally used to refer to a man who paid a lot of attention to his appearance, but became more broadly used to describe any man as part of African American Vernacular English in the 1960s. It was subsequently appropriated by white men in the 1980s, particularly on the West Coast, and has since grown to be used broadly by white men and women as a noun as well as a discourse marker.
+
+When used as a discourse marker, it carries the connotation that the speaker believes that they are speaking to a man; however, as both a discourse marker and a noun, it is frequently used without regard for the gender or pronouns of listeners.
+
+## Impact
+
+Regardless of the intent of the speaker, prefacing a statement with "dude" risks misgendering listeners and perpetuating a culture that favors masculinity as the baseline for gender. Dude users commonly defend its usage aggressively by claiming that they use it in a gender neutral way and that anyone negatively impacted by its usage is wrong for not accepting that. The ubiquity of that argument often deters those who are negatively impacted from bringing attention to the usage of dude.
```

Comment:
```suggestion
Regardless of the intent of the speaker, prefacing a statement with "dude" risks misgendering its audience and perpetuating a culture that favors traditional masculinity as the baseline for gender. Proponents of using "dude" commonly defend its usage aggressively by claiming that they use it in a gender-neutral way (which is not possible) and that anyone negatively impacted by its usage is wrong for not accepting that and chooses to apply the gendered interpretation. The argument fails to acknowledge that society dictates and reinforces gender, regardless of the individual interpretation. The ubiquity of that argument often deters those who are negatively impacted from bringing attention to the usage of "dude" as their experiences are often [gaslighted](/definitions/gaslighting).
```

Reply:
I did not realize that suggestions needed to be batched before committing the first one, but I have added this as well in a separate commit and attempted to provide proper attribution.